src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,45 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+  private Set<Integer> numbers;
+
+  @BeforeEach
+  void setUp() {
+    numbers = new HashSet<>();
+    numbers.add(1);
+    numbers.add(1);
+    numbers.add(2);
+    numbers.add(3);
+  }
```

Comment:
`@BeforeEach` 활용 좋습니다 👍 
아래처럼 표현도 가능하니 참고만 해주세요~

```suggestion
  @BeforeEach
  void setUp() {
    numbers = new HashSet<>(Arrays.asList(1, 1, 2, 3));
  }
```

Reply:
안녕하세요 경준님 피드백 감사합니다.

성능상의 차이점은 있을까요?